webapp-django/crashstats/supersearch/form_fields.py

```diff
@@ -116,6 +118,12 @@ def value_to_string(self, value):
         except AttributeError:  # when value is None
             return value
 
+    def strptime(self, value, format):
```

Comment:
I can reproduce that behavior, and I do not understand what is happening there. The thing is, it works fine in unit and manual tests, so that really confuses me! 

I've added the timezone like you suggested though.